Library/Homebrew/cask/audit.rb

```diff
@@ -550,6 +554,33 @@ def check_download
       add_error "download not possible: #{e}"
     end
 
+    def check_signing
+      return if !signing? || cask.url.blank?
+
+      odebug "Auditing signing"
+      artifacts = cask.artifacts.filter { |k| k.is_a?(Artifact::Pkg) || k.is_a?(Artifact::App) }
+
+      puts artifacts
```

Comment:
```suggestion
```

I assume this is just a debug message, but if not, I don't think we want anything to be printed unless there's an issue